docs/source/general/guides/drift_guide.rst

```diff
@@ -181,6 +185,32 @@ For code examples, see `here <#tabular-checks>`__
 All of these checks appear also in the `deepchecks interactive demo <https://checks-demo.deepchecks.com>`__, where you can
 insert corruption into the data and see the checks at work.
 
+Text (NLP) Data
+---------------
+
+Regarding `data <#data-drift>`__ or `concept drift <#concept-drift>`__
```

Comment:
did you build docs and check if all links work?
+
Can you add a pdf of this page? so it can be easily reviewed within the context

Reply:
submitting review, will continue it on pdf

Reply:
Added PDF.
About links - checked locally and made sure those that don't work locally (ref) are phrased as in other places. Will make sure with makedocs before merging.